log_drains.go

```diff
@@ -0,0 +1,71 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type LogDrainsService interface {
+	LogDrainsList(app string) ([]LogDrain, error)
+	LogDrainAdd(app string, params LogDrainAddParams) (*LogDrain, error)
+	LogDrainRemove(app string, id string) error
+}
+
+var _ LogDrainsService = (*Client)(nil)
+
+type LogDrain struct {
+	AppID string `json:"app_id"`
+	URL   string `json:"url"`
+}
+
+type LogDrainsRes struct {
+	LogDrains []LogDrain
+}
+
+type LogDrainRes struct {
+	LogDrain LogDrain `json:"log_drain"`
+}
+
+func (c *Client) LogDrainsList(app string) ([]LogDrain, error) {
+	var logDrainsRes []LogDrain
+	err := c.ScalingoAPI().SubresourceList("apps", app, "log_drains", nil, &logDrainsRes)
+	if err != nil {
+		return nil, errgo.Mask(err)
+	}
+	return logDrainsRes, nil
+}
+
+type LogDrainAddParams struct {
+	AppID string `json:"app_id"`
+	URL   string `json:"url"`
+}
+
+func (c *Client) LogDrainAdd(app string, params LogDrainAddParams) (*LogDrain, error) {
+	var logDrainRes LogDrainRes
+	err := c.ScalingoAPI().SubresourceAdd("apps", app, "log_drains", LogDrainRes{
+		LogDrain: LogDrain{
+			AppID: params.AppID,
+			URL:   params.URL,
+		},
+	}, &logDrainRes)
+	if err != nil {
+		return nil, errgo.Mask(err)
+	}
+	return &logDrainRes.LogDrain, nil
+}
+
+func (c *Client) LogDrainInfo(app, id string) (*LogDrain, error) {
```

Comment:
I'm not sure we will ever need this route :thinking: Do you think of a use case?

If you plan to keep it, please rename the method to match what we do in other services:
```suggestion
func (c *Client) LogDrainShow(app, id string) (*LogDrain, error) {
```